dmoj/graders/interactive.py

```diff
@@ -0,0 +1,75 @@
+from dmoj.graders.standard import StandardGrader
+
+class WrongAnswer(BaseException):
+    pass
+
+class Interactor(object):
+    def __init__(self, process):
+        self.process = process
+        self._is_closed = False
+        self._tokens = None
+
+    def read(self):
+        return self.process.stdout.read()
+
+    def readln(self):
+        return self.process.stdout.readline()
```

Comment:
I may be wrong, but I think this will return a newline that most people probably don't care about. `rstrip` would come in handy.